apps/studio/src/features/dashboard/components/ResourceTable/ResourceTable.tsx

```diff
@@ -54,19 +36,33 @@ const columns = [
     header: () => <TableHeader>Actions</TableHeader>,
     cell: ({ row }) => (
       <ResourceTableMenu
-        title={row.original.name}
+        title={row.original.title}
         resourceId={row.original.id}
-        type={row.original.type}
+        type={row.original.mainBlobId ? "page" : "folder"}
```

Comment:
i think this should be `mainBlobId || draftBlobId` - this is because if a site has never previously published anything, this `mainBlobId`  will be empty